### DISCUSSION TITLE
GLClient is not defined

### DISCUSSION THREAD

**Discussion by robcasares** [globaleaks/globaleaks-whistleblowing-software#3480] [Q&A]
Hello, I have created a script to change some parts of the main page, but it gives me the following error "Uncaught ReferenceError: GLClient is not defined" Is there something I have to install or something I am missing, I have renamed the script with the name "1", the CSS file works perfectly for me.

Global leaks version is 4.11.5
![Captura](https://github.com/globaleaks/GlobaLeaks/assets/136425776/0494389b-5032-4a5a-b43e-97fe39be0264)

![Captura2](https://github.com/globaleaks/GlobaLeaks/assets/136425776/62bf2e21-81e4-4c10-b64c-b63ac88f3a83)

**Comment by evilaliv3** [comment]
Thank you for the question.

The variable GLClient has been renamed to GL

**Reply by robcasares** [reply]
thank you very much, it works
